src/test/java/org/primeframework/mvc/GlobalTest.java

```diff
@@ -2140,6 +2141,31 @@ public void post_scopeStorageInBaseClass() throws Exception {
         .assertContextAttributeNotNull("contextObject");
   }
 
+  @Test
+  public void post_unsupported_multipleParameters() throws Exception {
+    // Cannot jam an array into a non-array data type
+    // - We don't currently have an easy way to capture the exception in the execute thread.
+
+    // UUID, two values, no dice.
+    // - We will get a 200, watch the console for the log output.
```

Comment:
Does this mean that there is nothing that can be asserted on the test? You just see the exception in the logs?

Reply:
Yes. I ended up re-working it so that I can assert on the last exception.